src/profile-logic/profile-data.js

```diff
@@ -2862,16 +2862,16 @@ export function getOrCreateURIResource(
   originToResourceIndex.set(origin, resourceIndex);
```

Comment:
while you're here, what do you think about inline the line 2861 `const originStringIndex = stringTable.indexForString(origin);` to line 2866? This value doesn't seem to be used elsewhere (is that normal?).

Reply:
Huh, that is indeed a bit weird. However, the value is only unused in the `host == null` case, and in that case `origin == scriptURI`, so then the call `stringTable.indexForString(scriptURI)` will just evaluate to the same string index. But yes it's weird, I'll fix it.